test/test_skvbc_persistence.py

```diff
@@ -114,3 +114,43 @@ def test_view_change_transitions_safely_without_quorum(self):
                                 # success!
                                 nursery.cancel_scope.cancel()
 
+    def test_read_written_data_after_restart_of_all_nodes(self):
+        """
+        This test aims to validate the blockchain is persistent
+        (i.e. the data is still available after restarting all nodes)
+        1) Write a key-value entry to the blockchain
+        2) Stop all replicas
```

Comment:
Or rather, "Restart all replicas (stop all, followed by start all)".

We should be careful to note that all of the replicas have been stopped because otherwise state transfer could kick in and impact the behavior of the test.

Regards,
Teo

Reply:
Fixed. Thanks!